### PR TITLE
Now support for .NET

### PR DESCRIPTION
**Pull request by draitchbar**
Added a C# file

- [ ] Check that it works.

### CODE REVIEW

**Comment by mikkelb-forecast** [comment]
This looks good, keep going.

File: main.cs [hunk 1]
```diff
@@ -0,0 +1 @@
+import net
```
Comment (mikkelb-forecast) [inline main.cs:1]: This looks good.

File: test.cs [hunk 2]
```diff
@@ -0,0 +1 @@
+import pandas
```
Comment (mikkelb-forecast) [inline test.cs:1]: Check spelling.

**Review by mikkelb-forecast — APPROVED** [review]
This all looks good, do check spelling

**Comment by draitchbar** [comment]
Keep both changesets.

**Pull request merged**